### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by KostyaSha**
While working on #467

### CODE REVIEW

File: src/main/java/com/github/dockerjava/core/DockerClientConfig.java [hunk 1]
```diff
@@ -164,16 +164,11 @@ private static Properties overrideDockerPropertiesWithSettingsFromUserHome(Prope
         Properties overriddenProperties = new Properties();
         overriddenProperties.putAll(p);
 
-        final File usersDockerPropertiesFile = new File(systemProperties.getProperty("user.home"), "."
-                + DOCKER_JAVA_PROPERTIES);
+        final File usersDockerPropertiesFile = new File(systemProperties.getProperty("user.home"),
+                "." + DOCKER_JAVA_PROPERTIES);
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/core/DockerClientConfig.java]: placed filename on one line to not break my brain

**Comment by KostyaSha** [comment]
@marcuslinke please review

**Comment by marcuslinke** [comment]
@KostyaSha Does not compile

**Comment by KostyaSha** [comment]
fixed

**Pull request merged**